NOTICE

```diff
@@ -1,5 +1,5 @@
-Apache Fluo
-Copyright 2017 The Apache Software Foundation.
+Apache Fluo Uno
+Copyright 2018-2019 The Apache Software Foundation.
```

Comment:
Can just put 2019 here. There's no added value to putting a range.

Reply:
Fixed now